Features.md

```diff
@@ -14,27 +14,308 @@
+---
+#### Author Name 
+
+This check determines whether the document includes the name of the author.
+
+The person, organization, or the tool that created the SBOM must be specified as the Author.
+
+***Remediation***
+Check and populate the following fields with the name of the person, organization, or tool creating the SBOM.
+- CycloneDX field: [metadata:authors](https://cyclonedx.org/docs/1.4/json/#metadata_authors)
+- SPDX field: [Creator](https://spdx.github.io/spdx-spec/v2.3/document-creation-information/#68-creator-field)
+
+---
+#### Timestamp 
+
+This check determines if the document includes the timestamp of its creation.
+
+The timestamp can be used to determine when the SBOM was created relative to the software itself.
+
+***Remediation steps***
+- Check and populate the following fields with the timestamp of the SBOM document.
+- CycloneDX field: [metadata:timestamp](https://cyclonedx.org/docs/1.4/json/#metadata_timestamp)
+- SPDX field: [Created](https://spdx.github.io/spdx-spec/v2.3/document-creation-information/#69-created-field)
+
+---
+#### Relationship among Components 
+
+This check determines if the document describes the relationship among included components.
+
+The dependency relationship can be critical in determining the order of inclusion and updates.
+
+***Remediation***
+- Check and populate the following fields with the relationship of components in the SBOM.
+- CycloneDX field: [dependencies](https://cyclonedx.org/docs/1.4/json/#dependencies)
+- SPDX field: [Relationship](https://spdx.github.io/spdx-spec/v2.3/relationships-between-SPDX-elements/#111-relationship-field)
+
+---
+### Category: Semantic
+---
+#### Component Checksum
+
+This check determines whether each component in the SBOM includes a valid checksum.
+
+A valid checksum can be used to independently identify the contents of the package among variations of the package.
+
+***Remediation***
+
+- Check and populate the following fields with the relationship of components in the SBOM.
+- CycloneDX field: [dependencies](https://cyclonedx.org/docs/1.4/json/#dependencies)
+- SPDX fields: [PackageChecksum](https://spdx.github.io/spdx-spec/v2.3/package-information/#710-package-checksum-field), [FileChecksum](https://spdx.github.io/spdx-spec/v2.3/file-information/#84-file-checksum-field)
+
+
+---
+#### Component License
+
+This check determines whether each component in the SBOM includes a valid license.
+
+A declared valid SPDX license is the key to evaluating any compliance risks.
+
+***Remediation steps***
+
+Check and populate the following fields with the relationship of components in the SBOM.
+- CycloneDX field: [component:licenses](https://cyclonedx.org/docs/1.4/json/#components_items_licenses)
+- SPDX fields: [PackageLicenseConcluded](https://spdx.github.io/spdx-spec/v2.3/package-information/#713-concluded-license-field), [LicenseConcluded](https://spdx.github.io/spdx-spec/v2.3/file-information/#85-concluded-license-field)
+
+---
+#### Required Fields
```

Comment:
at this point we only check metadata and components, in future we will expand it to files, etc

Reply:
I have marked all the Files checks to "Coming Soon"